drivers/include/ads1115.h

```diff
@@ -0,0 +1,216 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Baptiste Le Duc <baptiste.leduc38@gmail.com>
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+#pragma once
+
+/**
+ * @defgroup   drivers_ads1115 ADS1115 Analog-to-digital converter driver
+ * @ingroup    drivers_sensors
+ * @ingroup    drivers_saul
+ * @brief      I2C Analog-to-digital converter with programmable gain amplifier.
+ *             The device has four input channels: AIN0, AIN1, AIN2, and AIN3,
+ *             that can be selected by using @ref ads1115_set_ain_ch_input.
+ *
```

Comment:
```suggestion
 * @{
 *
```

Without the opening bracket, Doxygen seems to ignore the content of this file:
<img width="1568" height="899" alt="image" src="https://github.com/user-attachments/assets/9bea5291-2aa8-4765-876e-12947e9d2ab2" />